spec/kitchen/driver/gce_spec.rb

```diff
@@ -117,7 +118,8 @@
         zone_name: 'europe-west1-a',
         google_key_location: '/path/to/foo.p12',
         google_json_key_location: '/path/to/bar.json',
-        preemptible: true
+        preemptible: true,
+        auto_restart: true
```

Comment:
This just tests parameter override, so it will not affect the tests, however, you are right that this will make it confusing down the road. Let me think on how to refactor this carefully.